### DISCUSSION TITLE
Lost access to Authy 2FA and can no longer log in to Supabase

### DISCUSSION THREAD

**Discussion by raf723** [supabase/supabase#30125] [Questions]
Hi, I had to factory reset my phone and as a result, I've lost my Authy setup that I used to get 2FA codes to log in to my Supabase account. I know that it's a huge oversight on my part and I should have kept backup codes. I have a production app with thousands of users. Is there any way Supabase or Authy can intervene?

**Comment by GaryAustin1** [comment]
You can contact support, but also read this:
https://supabase.com/docs/guides/platform/multi-factor-authentication
